### PR TITLE
Some documentations

### PR DESCRIPTION
**Pull request by philippe2803**
Basically added examples for the main methods.

Plenty of commit called MAJ as I was trying rendering links properly on github in RST markup. Finally figure it out at the end.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,6 @@ dist: trusty
 language: python
 python:
 - '2.7'
-- '3.3'
```
Comment (SP-alex-muelas) [inline .travis.yml]: How come we remove python 3.3?

File: documentation/usage.rst [hunk 2]
```diff
@@ -0,0 +1,591 @@
+Sheetfu API usage
```
Comment (SP-alex-muelas) [inline documentation/usage.rst]: Amazing documentation! Very clear to understand for newcomers, and even I learnt some new things I didn't know (like the editor thing in the sa.create() xD)

Thanks a lot!

File: .travis.yml (hunk 1 above)
Comment (sp-philippe-oger) [reply .travis.yml]: Travis was failing for that version suddenlyÑ https://travis-ci.org/socialpoint-labs/sheetfu/builds/594308093.

The logs are saying: "google-api-python-client requires python3 version >= 3.4". I'm assuming this is a new dependency issue.

File: documentation/usage.rst (hunk 2 above)
Comment (sp-philippe-oger) [reply documentation/usage.rst]: Thanks for the feedback. 
If you're fine with it, could you please merge it?

Comment (SP-alex-muelas) [reply documentation/usage.rst]: Sure! If it's ok I'll do a "squash and merge" to avoid having all those "MAJ" commits in the commit-log =)

**Comment by sp-philippe-oger** [comment]
Just realized the documentation for the Table module needs to be done. I'll add it at some point this week.

File: documentation/usage.rst (hunk 2 above)
Comment (sp-philippe-oger) [reply documentation/usage.rst]: Yeah, that's a good idea.

**Comment by SP-alex-muelas** [comment]
I completely forgot (early in the morning xD), for future commits they should be done in the normalized format so that semantic releases can update and publish the version. Like:

docs: Add docs for Spreadsheet

In this case, as it's only documentation, it would not publish anything.

**Comment by sp-philippe-oger** [comment]
Completely forgot the semantic thingy. I'll remember next time.

**Pull request merged**